CMakeLists.txt

```diff
@@ -146,7 +156,7 @@ set_property(TARGET cpu_features PROPERTY POSITION_INDEPENDENT_CODE ${BUILD_PIC}
 target_include_directories(cpu_features
   PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/cpu_features>
 )
-if(PROCESSOR_IS_X86)
+if(PROCESSOR_IS_X86 OR PROCESSOR_IS_AARCH64)
```

Comment:
I know you're following the original code here, but it would make more sense to instead do just a single `if` here since the clauses are just about `APPLE`, e.g.:
```
if(APPLE AND (PROCESSOR_IS_X86 OR PROCESSOR_IS_AARCH64))
```

Reply:
Good point. I'm not great with Cmake yet. I'm sure there will be more changes, but currently it passes the build checks so I'll probably make the change locally and wait on it to submit until we get any more feedback.